presto-native-execution/Makefile

```diff
@@ -109,6 +109,3 @@ help:					#: Show the help messages
 	@cat $(firstword $(MAKEFILE_LIST)) | \
 	awk '/^[-a-z]+:/' | \
 	awk -F: '{ printf("%-20s   %s\n", $$1, $$NF) }'
-
-gen-pom-xml:
-	scripts/gen-pom-xml.sh
```

Comment:
Should we delete gen-pom-xml.sh as well?

Reply:
> Should we delete gen-pom-xml.sh as well?

yes, we should. let me do that